python/plugins/processing/algs/qgis/ui/InterpolationWidgets.py

```diff
@@ -313,8 +320,9 @@ def pixelSizeChanged(self):
 
     def rowsChanged(self):
         rows = self.mRowsSpinBox.value()
-        if rows <= 0:
+        if rows <= 0 or self.extent.isNull():
             return
+
         cell_size = self.extent.height() / rows
```

Comment:
Same comment as above (check .height() != 0 ?) -- is there a good place to add a method to avoid the duplication ?